package.json

```diff
@@ -17,8 +17,11 @@
     "test": "npm run lint && npm run test:once",
     "test:once": "jest --coverage",
     "test:watch": "jest --watch",
+    "prebuild": "rimraf dist tsconfig.tsbuildinfo",
```

Comment:
why is `rimraf` preferable to `rm -rf` ?

Reply:
Just a personal habit and normal convention in node community.